### PR TITLE
Added support for `[BRANCH_NAME]` in `title` input

### PR DESCRIPTION
**Pull request by orty**
Hi again @shayki5, I have another proposition for a feature which I could leverage. This PR will allow to use a token in the PR title, like so:

title: Merge master into [BRANCH_NAME]

This would produce a dynamic replacement of the token after the current target branch name. Very useful when working multiple target branches. 
I edited also the helpMarkup for the `title` input to give an hint to the user on this feature.

I am kinda coding happy and your extension really helped me a lot, I am glad to contribute to it :)

### CODE REVIEW

**Review by shayki5 — COMMENTED** [review]
Great feature! put attention that the value will be refs/heads/branch-name and not only the name, is it ok?

**Comment by orty** [comment]
Good advice, I will change that to reflect only the display name of the branch

**Comment by orty** [comment]
Hi @shayki5,

Just in case you missed it, I patched my PR according to your recommendation regarding the branch name :)

**Comment by shayki5** [comment]
Hi!
Thank you!
I already replied you:

> You need to remove the first / (should be refs/heads/).
> And check my last comment, in this case only the first branch will be repalced.
> You need to put the value in another variable (not in the $title).

**Review by shayki5 — COMMENTED** [review]
> Hi @shayki5,
> 
> Just in case you missed it, I patched my PR according to your recommendation regarding the branch name :)

Did you see my last comment?

**Comment by orty** [comment]
Hi, I am verry sorry I have been overwelmed by the job the last two weeks :/. 
I will do the changes you suggested me now that I am more available :)

**Comment by shayki5** [comment]
All good :) take your time.

File: task/createPullRequest.ps1 [hunk 1]
```diff
@@ -94,6 +94,7 @@ function RunTask {
         }
 
         foreach($branch in $targetBranches) {
+            $title = $title.Replace("[BRANCH_NAME]", $branch)
```
[the remark this answers is not in the document]
Comment (orty) [reply task/createPullRequest.ps1]: Done.

File: task/createPullRequest.ps1 [hunk 2]
```diff
@@ -94,7 +94,7 @@ function RunTask {
         }
 
         foreach($branch in $targetBranches) {
-            $title = $title.Replace("[BRANCH_NAME]", $branch)
+            $title = $title.Replace('[BRANCH_NAME]', $branch.Replace('/refs/heads/',''))
```
[the remark this answers is not in the document]
Comment (orty) [reply task/createPullRequest.ps1]: Now that I look at it, seems pretty obvious.. Should have seen it :)

**Comment by orty** [comment]
Hey :)

I think you will find it more accurate. Thx for your advices

**Review by shayki5 — APPROVED** [review]
Looks good now! I also testes it in my private project.
Thank you!

**Pull request merged**